src/notification-banner/notification-banner.styles.tsx

```diff
@@ -19,11 +19,19 @@ interface WrapperStyleProps {
 // =============================================================================
```

Comment:
is `notification-banner-label.tsx` still used / intended to be used? can either combine or remove that file

Reply:
nope it is no longer intended. the file is not present though. What do u mean?

Reply:
Yes it is no longer needed. Have removed